src/sidebar/components/test/filter-status-test.js

```diff
@@ -84,7 +84,7 @@ describe('FilterStatus', () => {
   context('(State 1): no search filters active', () => {
     it('should return null if filter state indicates no active filters', () => {
       const wrapper = createComponent();
-      assert.isEmpty(wrapper);
+      assert.equal(wrapper.children().length, 0);
```

Comment:
`FilterStatus` doesn't produce any direct HTML output, only child components. Currently all of those children are defined in the same module as `FilterStatus` and not mocked. So `assert.equal(wrapper.html(), '')` would actually work here. I'm being a little more strict/safe.